homeassistant/components/uvc/camera.py

```diff
@@ -199,3 +210,13 @@ def enable_motion_detection(self):
     def disable_motion_detection(self):
         """Disable motion detection in camera."""
         self.set_motion_detection(False)
+
+    async def stream_source(self):
+        """Return the source of the stream."""
+        caminfo = self._nvr.get_camera(self._uuid)
```

Comment:
This call does sync I/O. That will block the event loop which isn't allowed. Please schedule this call on the executor thread pool with `hass.async_add_executor_job`.